tools/asset-inventory/gae/config.yaml

```diff
@@ -13,19 +13,55 @@
 # limitations under the License.
 
 # Where the export is written to. A GCS path like gs://my-bucket-name/asset_export
-gcs_destination: gs://bmenasha-1-cai-export/asset_export
+# see import_pipeline_runtime_environment for another location to override this value.
+gcs_destination: <ENTER-BUCKET-URL>
 
 # Organization number (organizations/123) or project id (projects/id) or number (projects/123)
-export_parent: projects/bmenasha-1
+export_parent: <ENTER-PARENT>
 
 # BigQuery dataset to load to.
-import_dataset: cai_assets
+import_dataset: <ENTER-DATASET>
 
 # Location to write intermediary data to load from from. A GCS path like gs://my-bucket-name/asset_export/stage
-import_stage: gs://bmenasha-1-cai-export/asset_export_stage
+import_stage: <ENTER-STAGE>
 
 # Project id to run the dataflow job in when using the template or dataflow runner.
-import_dataflow_project: bmenasha-1
+import_dataflow_project: <ENTER-PROJECT>
+
+# When using the template runner. Supply values as a json representation of
+# RuntimeEnvironment:
+# https://cloud.google.com/dataflow/docs/reference/rest/v1b3/RuntimeEnvironment
+# If you are running the Dataflow workers in a shared VPC, set the VPC network
+# and subnet here.
+import_pipeline_runtime_environment: >
+  {
+    # if you don't get a warning about firewall rules preventing shuffle
```

Comment:
These comments should be moved to the troubleshooting section of the README, not burried in a config file. I would link out to the [docs on dataflow routes and firewall](https://cloud.google.com/dataflow/docs/guides/routes-firewall). I typically recommend they make a dataflow subnet with these rules and specify this when running their pipeline. Makes it easier to run with private ip as well.

Reply:
great. thanks for looking. 
I made the changes.